spyder/plugins/ipythonconsole.py

```diff
@@ -477,15 +477,42 @@ def setup_page(self):
         run_file_group.setLayout(run_file_layout)
         
         # ---- Advanced settings ----
+        # Enable Jedi completion in Ipyhton Console
+        ipy_jedi_group = QGroupBox(_("Jedi completion in Ipython console"))
+        ipy_jedi_label = QLabel(_("Enable Jedi based <tt>Tab</tt> completion in"
+                                "the Ipython console. E.g. completion of"
+                                "nested lits etc. This is similar to the "
+                                "greedy compiler, but without evaluating the code<br>"
+                                "<b>Attention:</b> This can slow down your console "
+                                "interaction when working with large dataframes!"))
+        ipy_jedi_label.setWordWrap(True)
+        ipy_jedi_box = newcb(_("Use Jedi Completer in Ipython"), "ipy_jedi_completer",
+                           tip="<b>Warning</b>: This can slow down your console"

```

Comment:
Fix alignment; should be a few spaces right to align one col right of the opening paren of ``newcb``. Also, doesn't this need to be translated too, (with ``_``) or do we not translate our ``tip``s?

Reply:
Also, use the above wording.